src/views/blocks/tickets/submit-button-modal.php

```diff
@@ -20,8 +20,11 @@
  */
 
 /* translators: %s is the event or post title the tickets are attached to. */
-$title       = sprintf( _x( '%s Tickets', 'Modal title. %s: event name', 'event-tickets' ), get_the_title() );
-$button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
+/* translators: %s: Tickets label */
+$title       = sprintf( _x( '%s %s', 'Modal title. %s: event name', 'event-tickets' ), get_the_title(), tribe_get_ticket_label_plural( 'event-tickets' ) );
```

Comment:
* :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title
* :warning: Equals sign not aligned correctly; expected 1 space but found 7 spaces
* :no_entry_sign: Multiple placeholders should be ordered. Expected '%1$s, %2$s', but got %s, %s.

<sub>hash: ceabb0a6e40dbd7d0899558310fd08b9</sub>